### ISSUE TITLE
Pkg output not showing in Jupyter notebook

### ISSUE THREAD

**Issue by crstnbr** [JuliaLang/IJulia.jl#1008]
<img width="1460" alt="Screenshot 2021-06-10 at 09 01 30" src="https://user-images.githubusercontent.com/187980/121479567-7517c180-c9ca-11eb-93e9-659a1cf95a5c.png">

I reinstalled Julia 1.6, tested everything with a clean `.julia` and only those packages:

```Julia
(@v1.6) pkg> st
      Status `~/.julia/environments/v1.6/Project.toml`
  [7073ff75] IJulia v1.23.2
  [438e738f] PyCall v1.92.3
```

Once I set `Pkg.DEFAULT_IO[] = stdout` everything is back to normal (until I restart the session).

**Comment by fredrikekre** [comment]
https://github.com/JuliaLang/Pkg.jl/pull/2543

**Comment by crstnbr** [comment]
Thanks for pointing out the relevant issue. Given this is fixed over at Pkg.jl, am I right in assuming that this will (only) be fixed in Julia 1.6.2?
